README.rst

```diff
@@ -124,18 +124,33 @@ sites)::
   ./manage.py lms collectstatic
   ./manage.py cms collectstatic
 
```

Comment:
commit should be `docs:` not `feat:`

Reply:
It's feat because of the change to devstack.py which adds new behavior being able to take a setting from the config file but happy to split them if you'd like.

Reply:
Fair enough